include/envoy/http/codec.h

```diff
@@ -197,7 +197,7 @@ struct Http2Settings {
   // initial value from HTTP/2 spec, same as NGHTTP2_DEFAULT_HEADER_TABLE_SIZE from nghttp2
   static const uint32_t DEFAULT_HPACK_TABLE_SIZE = (1 << 12);
   // no maximum from HTTP/2 spec, use unsigned 32-bit maximum
-  static const uint32_t MAX_HPACK_TABLE_SIZE = (1UL << 32) - 1;
+  static const uint32_t MAX_HPACK_TABLE_SIZE = 0xFFFFFFFF;
```

Comment:
You should be able to use the C++ numeric_limits for this:

```
#include <limits>
static const uint32_t MAX_HPACK_TABLE_SIZE = std::numeric_limits<uint32_t>::max();
```

Reply:
Done